collection.php

```diff
@@ -3,5 +3,28 @@
 $db = new PDO('mysql:host=DB;dbname=collection', 'root', 'password');
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query = $db->query('SELECT `name` FROM `characterinfo`;');
-$data = $query->fetchAll();
+$query = $db->query('SELECT * FROM `characterinfo` JOIN `houses` ON `characterinfo`.`house` = `houses`.`id`;');
+$hpcharacters = $query->fetchAll();
+
+?>
+
+<!DOCTYPE html>
+    <body>
+        <main>
+            <h1>Harry Potter Characters</h1>
+            <div>
+
```

Comment:
All looks good :) just maybe consider removing the wasted space (eg lines 16, 18, 24, 26) mike likes it to be more compressed rather than spaced out.